checkers/testdata/octalLiteral/positive_tests.go

```diff
@@ -76,7 +75,12 @@ func (os *OpenServer) Init(x int) {
 	os.x = x
 }
 
-func warningsOs() {
+func warningsFs() {
+	/*! use new octal literal style, 0o555 */
+	_ = fs.FileMode(0555)
+}
```

Comment:
Nothing changed with this func, I simply re-ordered it above the `warningsOs*` funcs for alphabetical order/consistency with `negative_tests.go`.